### ISSUE TITLE
Support for C# 6.0

### ISSUE THREAD

**Issue by Carpi68** [Visual-Stylecop/Visual-StyleCop#72]
I tried the new getter-only auto-property feature from C# 6.0.

This works:
`private static string MyString { get; } = @"Foo";`

But if I try to add multi line text like this:

```
private static string MyString`
    {
        get;
    } 
    = @"Foo 
    bar";
```

Then I'm getting `SA0102 : CSharp.CsParser : A syntax error has been discovered in file xxx on line 42.`

Possible workaround:
This is OK:
`private static string MyString => @"Foo";`
And also this is OK
``
`private static string MyString => 
    @"Foo
    bar";

```
```

**Comment by haison8x** [comment]
> var test = $"string\"";

This causes a syntax error

SA0102 : CSharp.CsParser : A syntax error has been discovered in file

**Comment by AdjutantML** [comment]
Yes please, support for auto property initializers would be awesome.
Currently the only reason why we havn't moved our projects to C# 6 yet.
